lts.json

```diff
@@ -14,5 +14,5 @@
       "ember-d3": "0.2.0"
```

Comment:
I'm not sure what these package groups are supposed to mean and why some seem to be empty. Could you explain it a bit?